x/rewards/keeper/providers.go

```diff
@@ -12,7 +14,18 @@ import (
 
 const DAY_SECONDS = 60 * 60 * 24
 
-func (k Keeper) AggregateRewards(ctx sdk.Context, provider, chainid string, adjustment sdk.Dec, rewards math.Int) {
+func (k Keeper) AddIprpcCu(ctx sdk.Context, provider string, chainID string, cu uint64) {
```

Comment:
can you call this AggregateCU?

Reply:
done. see https://github.com/lavanet/lava/commit/2a0f469e9a10c54367dbd0ecfdaa09dc8370ee1e